R/volcano_protti.R

```diff
@@ -72,11 +74,11 @@ volcano_protti <- function(data, grouping, foldchange, significance, method, pro
       labs(
         title = title,
         x = x_axis_label,
-        y = "-log10(p-value)"
+        y = y_axis_label
       ) +
       geom_hline(yintercept = -1 * log10(significance_cutoff), linetype = "dashed") +
-      geom_vline(xintercept = log2(foldchange_cutoff), linetype = "dashed") +
-      geom_vline(xintercept = -1 * log2(foldchange_cutoff), linetype = "dashed") +
+      geom_vline(xintercept = foldchange_cutoff, linetype = "dashed") +
+      geom_vline(xintercept = -1 * foldchange_cutoff, linetype = "dashed") +
```

Comment:
did you correct it the wrong way around? you should have added log2 to the other parts instead of removing it from these. Because we do not provide log2 fold change values but just fold change values.

Reply:
That's true. I think I got confused there. Or should we add log2 fold changes? IDK what is easier to use then. Do you by default calculate log2(FC) because I do

Reply:
yes my fold changes are log2 fold changes by default. But Viviane made the comment that it is confusing that we use a cutoff that is log2 so we changed it to be not log2.